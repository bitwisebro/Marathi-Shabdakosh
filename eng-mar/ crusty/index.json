{
  "Key": " crusty",
  "Meaning1": "जाड कवच असलेला "
}