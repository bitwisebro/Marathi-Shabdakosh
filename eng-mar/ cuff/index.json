{
  "Key": " cuff",
  "Meaning1": "चपराक"
}