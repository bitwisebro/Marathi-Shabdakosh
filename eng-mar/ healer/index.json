{
  "Key": " healer",
  "Meaning1": "रोग बरा करणारे"
}