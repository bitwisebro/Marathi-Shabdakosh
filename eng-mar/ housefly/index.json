{
  "Key": " housefly",
  "Meaning1": "माशी "
}