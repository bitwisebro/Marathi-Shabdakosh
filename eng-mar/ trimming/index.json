{
  "Key": " trimming",
  "Meaning1": "झालर लेस फीत इ.सजावट"
}