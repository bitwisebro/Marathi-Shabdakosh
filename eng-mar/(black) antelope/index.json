{
  "Key": "(black) antelope",
  "Meaning1": "काळवीट"
}