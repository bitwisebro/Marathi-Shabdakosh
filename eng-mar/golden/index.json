{
  "Key": "golden",
  "Meaning1": "सोन्याचा बनवलेला"
}